microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -251,7 +251,7 @@ If your extension displays ads, all content displayed must conform to the advert
 
 #### 1.10.4 Promotions
 
```

Comment:
closely related (in same sentence): the Merged [PR 2457](https://github.com/MicrosoftDocs/edge-developer/pull/2457) changed from:
```
[Microsoft Creative Specifications Policy](https://about.ads.microsoft.com/solutions/ad-products/display-advertising/creative-specs)
```
https://about.ads.microsoft.com/solutions/ad-products/display-advertising/creative-specs
to:
```
[Microsoft Creative Specifications Policy](https://about.ads.microsoft.com/policies/creative-specs)
```
https://about.ads.microsoft.com/policies/creative-specs
Destination page title: "Creative specs"
Destination tab title: "Creative specifications - Microsoft Advertising"
in section https://learn.microsoft.com/microsoft-edge/extensions-chromium/store-policies/developer-policies#1104-promotions